### PR TITLE
Introduce GraphSettings struct

### PR DESCRIPTION
**Pull request by AaronVanGeffen**
Replaces a bunch of consecutive loco_globals with one.

### CODE REVIEW

File: src/OpenLoco/src/Windows/CompanyList.cpp [hunk 1]
```diff
@@ -27,34 +27,43 @@ using namespace OpenLoco::Literals;
...
+        uint32_t dword_113DD8A;        // 0x0113DD8A
+        uint32_t dword_113DD8E;        // 0x0113DD8E
+        uint8_t pad_113DD92[7];        // 0x0113DD92
+        uint8_t byte_113DD99;          // 0x0113DD99
+        uint16_t itemId[32];           // 0x0113DD9A
+    };
+#pragma pack(pop)
+
+    static_assert(sizeof(GraphSettings) == 0x0113DD9A + 64 - 0x0113DC7A);
+    static loco_global<GraphSettings, 0x0113DC7A> _graphSettings;
```
Comment (ZehMatt) [inline src/OpenLoco/src/Windows/CompanyList.cpp:66]: Is this still required to be interop?

Comment (AaronVanGeffen) [reply src/OpenLoco/src/Windows/CompanyList.cpp:66]: Yes, the actual graph plotting in `drawGraph` hasn't been implemented yet! I'd like the code for it to live in another file, though. This kind of struct should allow us to  bridge that gap, and to easily add graphs to other areas in the future.

Comment (ZehMatt) [reply src/OpenLoco/src/Windows/CompanyList.cpp:66]: Oh right.

File: src/OpenLoco/src/Windows/CompanyList.cpp [hunk 2]
```diff
@@ -27,34 +27,43 @@ using namespace OpenLoco::Literals;
...
-    static loco_global<uint16_t, 0x0113DD82> _graphXAxisLabelIncrement;
-    static loco_global<uint16_t, 0x0113DD84> _graphYLabel;
-    static loco_global<uint32_t, 0x0113DD86> _dword_113DD86;
-    static loco_global<uint32_t, 0x0113DD8A> _dword_113DD8A;
-    static loco_global<uint32_t, 0x0113DD8E> _dword_113DD8E;
-    static loco_global<uint8_t, 0x0113DD99> _byte_113DD99;
-    static loco_global<uint16_t[32], 0x0113DD9A> _graphItemId;
+
+#pragma pack(push, 1)
+    struct GraphSettings
```
Comment (ZehMatt) [inline src/OpenLoco/src/Windows/CompanyList.cpp:34]: We should split graph parameters and graph data after we implemented the drawing, this bundles too much in one struct. For now we can leave it be.

**Review by ZehMatt — APPROVED** [review]

File: src/OpenLoco/src/Windows/CompanyList.cpp (hunk 2 above)
Comment (AaronVanGeffen) [reply src/OpenLoco/src/Windows/CompanyList.cpp:34]: Agreed. I'd also like to move it to e.g. Graph.h and put the plotting code in an accompanying unit file, e.g. Graph.cpp.

**Pull request merged**